src/wasm.h

```diff
@@ -1427,15 +1427,15 @@ class Function {
   }
 };
 
+enum class Kind {
```

Comment:
`GlobalKind` sounds like a good idea. Yeah, `Table` and `Memory` do fit in here (although globalopt won't do much with them until we have multiple tables and memories). So `GlobalKind` it is?